internal/broker/bind_create_test.go

```diff
@@ -352,6 +409,10 @@ func kubeconfigClient(t *testing.T, kubeconfig string) *kubernetes.Clientset {
 func verifyResponse(t *testing.T, response *http.Response) domain.Binding {
 	require.Equal(t, http.StatusCreated, response.StatusCode)
 
+	return unmarshal(t, response)
```

Comment:
Kind of misleading function name - verifyResponse returns `domainBinding` so main responsibility is extracting binding from the response.